src/frontend/src/lib/styles/global/colors.scss

```diff
@@ -4,22 +4,16 @@
 	--color-white: theme(colors.white);
 	--color-grey: theme(colors.grey);
 	--color-dust: theme(colors.dust);
-	--color-light-blue: theme(colors.light-blue);
```

Comment:
this one is not removes from Tailwind config because it is used as `bg-light-blue` in the code.